### ISSUE TITLE
Question about the `annotators_config` and `reference_outputs` in alpaca_eval

### ISSUE THREAD

**Issue by AIR-hl** [princeton-nlp/SimPO#55]
I would like to determine if the default configuration `annotators_config=weighted_alpaca_eval_gpt4_turbo` and `reference_outputs=gpt4_turbo ` used when evaluating with alpaca_eval. Thank you!

**Comment by xiamengzhou** [comment]
Yes, we used the default AlpacaEval 2 configuration for these two entries!
